static/js/publisher/release/actions/availableSelect.js

```diff
@@ -0,0 +1,8 @@
+export const SET_AVAILABLE_SELECT = "SET_AVAILABLE_SELECT";
```

Comment:
Can we call these `*_AVAILABLE_REVISIONS_*` and `*AvailableRevisions*` throughout? `AVAILABLE` isn't very descriptive at a glance IMO.

Reply:
Ok, good point. I didn't want to make the variable names too long, but I guess readability is more important. Fixed.